Sources/LiveKit/Publications/RemoteTrackPublication.swift

```diff
@@ -197,6 +197,28 @@ public class RemoteTrackPublication: TrackPublication {
         shouldComputeVideoViewVisibilities()
     }
     #endif
+
+    // MARK: - Suspend/Resume
+
+    @discardableResult
+    internal override func suspend() -> Promise<Void> {
+        // do nothing if already disabled
+        guard enabled else { return Promise(()) }
+        let newSettings = trackSettings.copyWith(enabled: false)
+        return send(trackSettings: newSettings).then {
+            self.suspended = true
+        }
+    }
+
+    @discardableResult
+    internal override func resume() -> Promise<Void> {
+        // do nothing if was not suspended
+        guard suspended else { return Promise(()) }
+        let newSettings = trackSettings.copyWith(enabled: true)
```

Comment:
what if it was disabled by the user, would this now be enabling it?

Reply:
if disabled by user, `suspended` will be false so nothing should happen.

Reply:
going into background `suspend()` gets called and back to foreground `resume()` gets called but
if user does some own logic it might conflict. So maybe this can be an option.